example/lib/main.dart

```diff
@@ -31,6 +32,16 @@ class MyApp extends StatelessWidget {
                 decoration: InputDecoration(hintText: 'List type'),
               ),
 
+              /// sample1
+              /// - list with updateHistoryItemDateTime
+              InputHistoryTextField(
+                historyKey: "05",
+                listStyle: ListStyle.List,
+                onHistoryItemSelected: (value) => print(value),
+                updateSelectedHistoryItemDateTime: true,
+                decoration: InputDecoration(hintText: 'List type'),
```

Comment:
`List type` is duplicated. please different `hintText`.

```suggestion
                decoration: InputDecoration(hintText: 'List type(update in descending order)'),
```